res/UseNode5.ps1

```diff
@@ -1,18 +0,0 @@
-$ErrorActionPreference='Stop'
```

Comment:
Are we sure we will never need this?

Reply:
It was only used in CI and we don't need to run against Node 5 anymore